temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -212,6 +212,53 @@ message DeleteNamespaceResponse {
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
 
+message FailoverNamespaceRequest {
+    // The namespace to failover.
+    string namespace = 1;
+    // The target region id to failover.
+    string target_region_id = 2;
+    // The id to use for this async operation.
+    // Optional, if not provided a random id will be generated.
+    string async_operation_id = 3;
+}
+
+message FailoverNamespaceResponse {
+    // The async operation.
+    temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
+}
+
+message GetNamespaceFailoverHistoryRequest {
```

Comment:
Remove for now. Follow up with @jumbogo about expose those fields from audit logging.